docs/external_system.md

```diff
@@ -99,33 +99,33 @@ use externally-accessible API endpoints exposed by CSM.
       csm-api-client@git+https://github.com/Cray-HPE/python-csm-api-client@release/1.1
       ```
 
-   1. (Optional) Confirm that `requirements.lock.txt` is modified as expected.
+   1. (Optional) (`venv`) Confirm that `requirements.lock.txt` is modified as expected.
 
       **Note:** For versions newer than 3.19, you will see both `cray-product-catalog`
       and `csm-api-client`. For version 3.19 and older, you will only see
       `cray-product-catalog`.
 
       ```screen
-      (venv) $ grep -E 'cray-product-catalog|csm-api-client' sat/requirements.lock.txt
+      $ grep -E 'cray-product-catalog|csm-api-client' sat/requirements.lock.txt
       ./cray-product-catalog
       csm-api-client@git+https://github.com/Cray-HPE/python-csm-api-client@release/1.1
       ```
```

Comment:
I think we need to separate the command that the user executes from the output of that command now. Otherwise, once we remove the command prompt portion here (the `$ `), the reader won't be able to tell which part is a command to execute and which part is the output of the command.  E.g., something like this:

````
   ```sh
   grep -E 'cray-product-catalog|csm-api-client' sat/requirements.lock.txt
   ```

   The output should look like this:

   ```
   ./cray-product-catalog
   csm-api-client@git+https://github.com/Cray-HPE/python-csm-api-client@release/1.1
   ```
````

Then maybe also move the note from lines 104-106 down below the output. You might also check with how docs-csm handles distinguishing commands from the command output.